app/models/jobsModel.js

```diff
@@ -5,18 +5,51 @@ const db = require('../database/dbConfig.js')
  * scheduled_time is an optional field.
  * 
  * @param {"class_id" : integer, 
- *         "scheduled_time" : Date, 
+ *         "scheduled_time" : Date,
  *         "job_hash" : string} job 
  */
-async function insertJob(job){
+async function insert(job){
     try {
         await db('jobs')
-        .insert(job)
-    }catch(e){
-        console.error("Error: failed to insert job", e)
+              .insert(job);
+    } catch(e){
+        console.error("Error: failed to insert job", e);
+    }
+}
+
+/**
+ * Updates scheduled_time for jobs in the database with the same class_id.
+ * 
+ * @param {"class_id" : integer,
+ *         "scheduled_time" : Date} updatedJob 
+ */
+async function update(updatedJob){
+    try {
+        await db('jobs')
+              .where({"class_id" : updatedJob.class_id})
```

Comment:
And also, why are we updating by class_id? there could be multiple jobs with the same class_id, and since you use a map function to iterate through them each iteration it will update every single jobs with the same class_id, suggestion: we can probably use just the id so it is unique